metadata-ingestion/src/datahub/ingestion/source/unity/proxy.py

```diff
@@ -251,6 +255,31 @@ def tables(self, schema: Schema) -> Iterable[Table]:
                     logger.warning(f"Error parsing table: {e}")
                     self.report.report_warning("table-parse", str(e))
 
+    def models(self, schema: Schema) -> Iterable[Model]:
```

Comment:
can we name these to ml_models to be more explicit what it is about?

Reply:
updated!